src/localstore/incremental/JasmineGraphIncrementalLocalStore.cpp

```diff
@@ -94,7 +94,7 @@ void JasmineGraphIncrementalLocalStore::addEdgeFromString(std::string edgeString
             }
         }
 
-        incremental_localstore_logger.log("Added successfully!", "Info");
+        incremental_localstore_logger.debug("Added successfully!");
```

Comment:
This log line does not carry any meaning. What do you mean by added successfully? what did you add successfully?

Reply:
changed the log into         `incremental_localstore_logger.debug("Edge (" + sId + ", " + dId + ") Added successfully!");`

Reply:
Ok